### PR TITLE
Enhanced duration format  and mosaic flags description 

### LINKED ISSUE #229: Sign a transaction of the namespace modal format uses 1 d 1 h 1 m，which is more friendly to display
image

### LINKED ISSUE #230: The 4 attributes in creating a new mosaic should be described
Unlimited duration / Transferable / Supply mutable / Restrictable

### PR DESCRIPTION
**Pull request by David0817**
* Add descriptions about mosaic flags (fix #230)

  * Change duration format to human friendly time periods (fix #229)

### CODE REVIEW

File: src/language/en-US.json [hunk 1]
```diff
@@ -186,6 +186,12 @@
   "transmittable": "Transferable",
   "variable_supply": "Supply mutable",
   "duration_permanent": "Unlimited duration",
+  "flags_supply": "The total supply must be in the range of 0 and 9,000,000,000,000,000 atomic units.",
+  "flags_divisibility": "Determines the decimal place to which the mosaic can be divided.The divisibility must be in the range of 0 and 6.",
```
Comment (evias) [inline src/language/en-US.json:190]: space after dot.

File: src/language/en-US.json [hunk 2]
```diff
@@ -186,6 +186,12 @@
...
+  "flags_transmittable": "If set to true,the mosaic can be transferred between arbitrary accounts.",
```
Comment (evias) [inline src/language/en-US.json:191]: - space after comma
- key should be `flags_transferable`.
- `If set to true, the mosaic can be transferred to arbitrary accounts.`

File: src/language/en-US.json [hunk 3]
```diff
@@ -186,6 +186,12 @@
...
+  "flags_variable_supply": "If set to true,the mosaic supply can change at a later point.",
```
Comment (evias) [inline src/language/en-US.json:192]: - space after comma
- `If set to true, the mosaic supply will be mutable.`

File: src/language/en-US.json [hunk 4]
```diff
@@ -186,6 +186,12 @@
...
+  "flags_restrictable": "If set to true,the mosaic creator can configure custom restrictions.",
```
Comment (evias) [inline src/language/en-US.json:193]: - space after comma
- `mosaic creator` should be `mosaic owner`

File: src/language/en-US.json [hunk 5]
```diff
@@ -186,6 +186,12 @@
...
+  "flags_duration_permanent": "If set to true or setting duration to 0, the mosaic will not expire.Otherwise, you can set the expiration time.",
```
Comment (evias) [inline src/language/en-US.json:194]: - `If set to true, the mosaic will never expire.`

File: src/language/en-US.json [hunk 6]
```diff
@@ -422,10 +428,10 @@
   "New_root_space": "New root space",
   "Subspace": "Sub-namespace",
   "Input_space_name": "Input namespace",
-  "namespace_tips_key_1": "1. The length of the root namespace is limited to 16 characters, the length of the secondary namespace is limited to 64 characters. Valid characters:",
-  "namespace_tips_key_2": "2. The first character of the namespace must use English letters. Values not allowed include:",
+  "namespace_tips_key_1": "1. The name must be unique in the network, and may have a maximum length of 64 characters. Valid characters:",
```
Comment (evias) [inline src/language/en-US.json:431]: `Valid characters:` should be `Accepted characters include: `

File: src/language/en-US.json [hunk 7]
```diff
@@ -422,10 +428,10 @@
...
+  "namespace_tips_key_2": "2. Values not allowed include:",
```
Comment (evias) [inline src/language/en-US.json:432]: `2. Following characters are not accepted: `

Comment (David0817) [reply src/language/en-US.json:432]: fix translation of the mosaic flags and namespace tips

File: src/components/TransactionDetails/NamespaceRegistration/NamespaceRegistration.vue [hunk 8]
```diff
@@ -38,13 +40,13 @@ export default class NamespaceRegistration extends Vue {
       'registrationType',
     )
     const duration: UInt64 = this.view.values.get('duration')
-
+    const blockGenerationTargetTime = this.networkConfiguration.blockGenerationTargetTime
```
Comment (evias) [inline src/components/TransactionDetails/NamespaceRegistration/NamespaceRegistration.vue:43]: Where is `this.networkConfiguration` model populated with a value? I don't see it mapped from Store.

Comment (fboucquez) [reply src/components/TransactionDetails/NamespaceRegistration/NamespaceRegistration.vue:43]: Missed:?
```
computed: {
    ...mapGetters({
      networkConfiguration: 'network/networkConfiguration',
    }),
  },
```

**Pull request merged**